setup.py

```diff
@@ -4,9 +4,9 @@
     long_description = fh.read()
 
 setuptools.setup(
-    name="great_expectations_airflow",
+    name="airflow-provider-great-expectations",
     version="0.0.1",
-    description="An Airflow operator for Great Expectations",
+    description="An Airflow provider for Great Expectations",
```

Comment:
```suggestion
    description="An Apache Airflow provider for Great Expectations",
```

We should follow http://www.apache.org/foundation/marks/#guidelines